README.md

```diff
@@ -1,43 +1,71 @@
-# onnx-cpu
+# `onnx-cpu` modular resource
 
-## Build it
+This module allows you to run ONNX models with the Viam Vision Service on Android.
 
-if you need to package it up in a module, do the following
+## Requirements
 
-```
-go build -o module cmd/module/main.go
-chmod a+x module
-tar -czf module.tar.gz module third_party/
-```
+Before configuring your ML model, you must [create a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine).
 
-`third_party` contains all of the `.so` files for the ONNX runntime. You can package only the one you need.
+## Build and run
 
-For Android, the Makefile will create the tar file.
+To use this module, follow these instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry) and select the `viam-labs:mlmodel:onnx-cpu` model from the [`onnx-cpu` module](https://github.com/viam-labs/onnx-cpu).
 
-You can then add the module to app.viam.com locally. the triplet is `viam-labs:mlmodel:onnx-cpu`
+## Configure your `onnx-cpu` sensor
 
-## Config
+Navigate to the **Config** tab of your machine's page in [the Viam app](https://app.viam.com/).
+Click on the **Services** subtab and click **Create service**.
+Select the `mlmodel` type, then select the `onnx-cpu:timesyncsensor` model.
+Click **Add module**, then enter a name for your ML model and click **Create**.
 
-- `model_path`, the full path to the ONNX file.
-- `label_path` (optional), the full path to the names of the category labels.
+On the new component panel, copy and paste the following attribute template into your ML model’s **Attributes** box:
 
+```json
+{
+  "model_path": "/path/to/onnx_file/detector_googlenet.onnx",
+  "label_path": "/path/to/labels.txt"
+}
 ```
+
+> [!NOTE]
+> For more information, see [Configure a Machine](https://docs.viam.com/manage/configuration/).
+
+### Attributes
+
+The following attributes are available for the `viam-labs:mlmodel:onnx-cpu` sensor:
+
+| Name    | Type   | Inclusion    | Description |
+| ------- | ------ | ------------ | ----------- |
+| `model_path` | string | **Required** | The full path to the ONNX file. |
+| `label_path` | string | Optional | The full path to the names of the category labels. |
+
+### Example configuration
+
+```json
     {
       "name": "onnx",
       "type": "mlmodel",
       "model": "viam-labs:mlmodel:onnx-cpu",
       "attributes": {
         "model_path": "/path/to/onnx_file/detector_googlenet.onnx"
-        "label_path": "/path/to/labels.txt
+        "label_path": "/path/to/labels.txt"
       }
     }
 ```
 
-## Use it with a vision service
+### Configure a vision service
```

Comment:
Transition sentence explaining why configuring vision service could be good beforehand

Reply:
Good point! Added!